### ISSUE TITLE
ECS Client has missing SettingName for the dualStackIPv6 option

### ISSUE THREAD

**Issue by jmenga-aset** [aws/aws-sdk-js-v3#5885]
### Checkboxes for prior research

- [X] I've gone through [Developer Guide](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide) and [API reference](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest)
- [X] I've checked [AWS Forums](https://forums.aws.amazon.com) and [StackOverflow](https://stackoverflow.com/questions/tagged/aws-sdk-js).
- [X] I've searched for [previous similar issues](https://github.com/aws/aws-sdk-js-v3/issues) and didn't find any solution.

### Describe the bug

The ECS Client [SettingName](https://github.com/aws/aws-sdk-js-v3/blob/2ed0a111fa38d3a3c64bbcfa281d726adf7ae469/clients/client-ecs/src/models/models_0.ts#L3960-L3970) enum is missing an option for the `dualStackIPv6` option (as indicated [here](https://github.com/aws/aws-sdk-js-v3/blob/2ed0a111fa38d3a3c64bbcfa281d726adf7ae469/clients/client-ecs/src/models/models_0.ts#L10180-L10184))

### SDK version number

@aws-sdk/client-ecs@3.529.1

### Which JavaScript Runtime is this issue in?

Node.js

### Details of the browser/Node.js/ReactNative version

v20.10.0

### Reproduction Steps

```javascript
import { SettingName } from '@aws-sdk/client-ecs'

console.log(SettingName)
```

### Observed Behavior

The following output is generated:

```
{
  AWSVPC_TRUNKING: 'awsvpcTrunking',
  CONTAINER_INSIGHTS: 'containerInsights',
  CONTAINER_INSTANCE_LONG_ARN_FORMAT: 'containerInstanceLongArnFormat',
  FARGATE_FIPS_MODE: 'fargateFIPSMode',
  FARGATE_TASK_RETIREMENT_WAIT_PERIOD: 'fargateTaskRetirementWaitPeriod',
  GUARD_DUTY_ACTIVATE: 'guardDutyActivate',
  SERVICE_LONG_ARN_FORMAT: 'serviceLongArnFormat',
  TAG_RESOURCE_AUTHORIZATION: 'tagResourceAuthorization',
  TASK_LONG_ARN_FORMAT: 'taskLongArnFormat'
}
```

### Expected Behavior

There should be a key/value pair for the value `dualStackIPv6`

e.g. 
```
{
  DUAL_STACK_IPV6: 'dualStackIPv6'
}
```

### Possible Solution

Update the `SettingName` [enum](https://github.com/aws/aws-sdk-js-v3/blob/2ed0a111fa38d3a3c64bbcfa281d726adf7ae469/clients/client-ecs/src/models/models_0.ts#L3960-L3970) to the following:

```
export const SettingName = {
  AWSVPC_TRUNKING: "awsvpcTrunking",
  CONTAINER_INSIGHTS: "containerInsights",
  CONTAINER_INSTANCE_LONG_ARN_FORMAT: "containerInstanceLongArnFormat",
  DUAL_STACK_IPV6: 'dualStackIPv6'
  FARGATE_FIPS_MODE: "fargateFIPSMode",
  FARGATE_TASK_RETIREMENT_WAIT_PERIOD: "fargateTaskRetirementWaitPeriod",
  GUARD_DUTY_ACTIVATE: "guardDutyActivate",
  SERVICE_LONG_ARN_FORMAT: "serviceLongArnFormat",
  TAG_RESOURCE_AUTHORIZATION: "tagResourceAuthorization",
  TASK_LONG_ARN_FORMAT: "taskLongArnFormat",
} as const;
```

**Comment by RanVaknin** [comment]
Hi @jmenga-aset ,

Thanks for reaching out. Its not clear if this was never modeled, or was removed. I'm curious, if you had to supply the string value directly instead of using the enum, does `dualStackIPv6` actually work? 

The reason why I'm asking is sometimes service teams remove enum values (in favor of introducing new parameters or deprecating old ones) and forget to remove the corresponding documentation. 

Thanks,
Ran~

**Comment by mixja** [comment]
Yes dualStackIPv6 is a documented setting and works - see [docs](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-account-settings.html#:~:text=Resource%20name%3A-,dualStackIPv6,-Amazon%20ECS%20supports)

**Comment by RanVaknin** [comment]
Hi there,

I have created an internal ticket (`P146481495`) with the EC2 service team for them to update their model with the missing enum.
Since the AWS SDKs are code-generated from those API models, the SDK team cannot add those directly. Instead, the service team itself would be the one needing to make the change on the backend.

Moving to cross SDK repo for tracking.

Thank,
Ran~
